playwright-tests/tests/infrastructure/proposal.spec.js

```diff
@@ -81,4 +88,65 @@ test.describe("Wallet is connected as admin", () => {
 
     await pauseIfVideoRecording(page);
   });
+
+  test("should show relevant users in mention autocomplete", async ({
```

Comment:
Is it possible to have a common test for all 3 instances here?

Reply:
Yes I planned that as a next step, after adding the functionality of kyc to the events committee portal as well as clean urls (which apparently already works on all portals)